molecule/scenario_resources/host_vars/fedora-29

```diff
@@ -1 +1,2 @@
 ansible_python_interpreter: /usr/bin/python3
+pulp_configure_firewall: none
```

Comment:
Thank you, but we should configure & test firewalld on every distro except CentOS 7 (because it's broken there.) Its a code path (in ansible-pulp) we should test.

Reply:
Wasn't clear enough...
It was broken in fedora 30 and it broke on Debian the moment, I switched the containers to the unprivileged mode. And I would rather keep that.

Reply:
I agree that privileged mode is very undesirable on your own system when you're running molecule, even though it's OK on our Travis CI VMs.

However, we still want to test the firewall code path. It is even a default enabled feature.

How about we give [the 2 capabilities](https://stackoverflow.com/a/44523870
) to the container instead?
Ansible module "[docker_container](https://docs.ansible.com/ansible/latest/modules/docker_container_module.html#parameters)" has a capabilities param, which you can specify in [molecule.yml](https://molecule.readthedocs.io/en/latest/configuration.html#docker)

Alternatively, you can do this work (unprivileged mode, capabilities, etc) in a separate commit, on a separate PR, that you rebase onto this branch. This branch/PR would keep the firewall enabled for everything but CentOS, and keep Privileged mode. You would not need to test this branch on your own system, just the other branch with the smaller commit on it.

Reply:
Thanks for the Capabilities idea!
Strangely enough it fixed iptables operations on centos and debian.
Still no luck on fedora. Honestly, i don't have the time to further investigate that now.

Reply:
Now that we have the firewall tested on at least 1 distro, that is sufficient to test the code path. Thank you.